### PR TITLE
[WIP] [BUG] Display title in plot_compare_evokeds when axes='topo'

### LINKED ISSUE #12191: plot_compare_evokeds doesn't display title when using axes='topo'
### Description of the problem

plot_compare_evokeds doesn't display title when using axes='topo'
Example: mne.viz.plot_compare_evokeds(evk, picks="eeg", axes="topo", title='TEST ERP')
Maybe this is not a bug and this was never intended. If so, maybe documentation should be updated?

### Steps to reproduce

```Python
sample_data_folder = mne.datasets.sample.data_path()
sample_data_raw_file = (
    sample_data_folder / "MEG" / "sample" / "sample_audvis_filt-0-40_raw.fif"
)

raw = mne.io.read_raw_fif(sample_data_raw_file)
events = mne.find_events(raw, stim_channel="STI 014")
event_dict = {
    "auditory/left": 1,
    "auditory/right": 2,
    "visual/left": 3,
    "visual/right": 4,
    "smiley": 5,
    "buttonpress": 32,
}
reject_criteria = dict(
    mag=4000e-15,  # 4000 fT
    grad=4000e-13,  # 4000 fT/cm
    eeg=150e-6,  # 150 µV
    eog=250e-6,
)  # 250 µV
epochs = mne.Epochs(
    raw,
    events,
    event_id=event_dict,
    tmin=-0.2,
    tmax=0.5,
    reject=reject_criteria,
    preload=True,
)
evk= epochs.average()
mne.viz.plot_compare_evokeds(evk, picks="eeg", axes="topo", title='TEST ERP')
```

### Link to data

_No response_

### Expected results

Plot should have 'TEST ERP' at the top of the figure. 

### Actual results

image

### Additional information

Platform             Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.31
Python               3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:40:35) [GCC 12.3.0]
Executable           /home/woess/mambaforge/envs/mne/bin/python
CPU                  x86_64 (12 cores)
Memory               11.3 GB

Core
├☑ mne               1.5.1
├☑ numpy             1.24.4 (OpenBLAS 0.3.24 with 12 threads)
├☑ scipy             1.11.3
├☑ matplotlib        3.8.0 (backend=module://matplotlib_inline.backend_inline)
├☑ pooch             1.7.0
└☑ jinja2            3.1.2

Numerical (optional)
├☑ sklearn           1.3.1
├☑ numba             0.57.1
├☑ nibabel           5.1.0
├☑ nilearn           0.10.2
├☑ dipy              1.7.0
├☑ openmeeg          2.5.6
├☑ pandas            2.1.1
└☐ unavailable       cupy

Visualization (optional)
├☑ pyvista           0.42.3 (OpenGL 3.3 (Core Profile) Mesa 21.2.6 via D3D12 (AMD Radeon(TM) Graphics))
├☑ pyvistaqt         0.0.0
├☑ vtk               9.2.6
├☑ qtpy              2.4.0 (PyQt5=5.15.8)
├☑ pyqtgraph         0.13.3
├☑ mne-qt-browser    0.5.2
├☑ ipywidgets        8.1.1
├☑ trame_client      2.12.5
├☑ trame_server      2.12.0
├☑ trame_vtk         2.5.9
├☑ trame_vuetify     2.3.1
└☐ unavailable       ipympl

Ecosystem (optional)
└☐ unavailable       mne-bids, mne-nirs, mne-features, mne-connectivity, mne-icalabel, mne-bids-pipeline

### PR DESCRIPTION
**Pull request by withmywoessner**
#### Reference issue
Example: Fixes #12191.

#### What does this implement/fix?
Add title to topoplot in `plot_compare_evokeds`

#### Additional information
Any additional information you think is important.

### CODE REVIEW

**Comment by withmywoessner** [comment]
The title is too small @larsoner. Any suggestions for formatting?
image

**Comment by withmywoessner** [comment]
My solution was not the most elegant, but the `plot_compare_evokeds `function is a but confusing at a glance.

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -3217,5 +3219,7 @@ def click_func(
     if cmap is not None:
         _draw_colorbar_pce(ax, _colors, _cmap, colorbar_title, colorbar_ticks)
     # finish
+    if topo_disp_title:
+        plt.suptitle(topo_title)
```
Comment (larsoner) [inline mne/viz/evoked.py:3223]: Prefer methods to the global pipeline approach
```suggestion
        ax.figure.suptitle(topo_title)
```

**Review by larsoner — COMMENTED** [review]
> The title is too small @larsoner. Any preference for how it should be formatted?

I think it's okay as is. Users can set `suptitle` font size via `rcParams` for example and we should just live with whatever it is

**Comment by withmywoessner** [comment]
Ready for merge @larsoner! The function didn't autocomplete when I typed it in in devel so I hope I specified it correctly.

**Comment by larsoner** [comment]
Can you go to https://app.circleci.com/pipelines/github/mne-tools/mne-python/21836/workflows/7c69bd2c-835a-428b-b60b-1fe65b988098/jobs/61390 and log in with GitHub? It will allow CircleCI to build your PRs

**Comment by withmywoessner** [comment]
I just did it now:
image

**Comment by withmywoessner** [comment]
Sorry, I had to merge again. I am not sure why the CircleCI test is failing. I logged in.

**Comment by withmywoessner** [comment]
I think there is something wrong with the tests. All of my PRs are failing. I ran them locally:
```
============================= test session starts ==============================
platform linux -- Python 3.11.6, pytest-7.4.3, pluggy-1.3.0
PyQt5 5.15.9 -- Qt runtime 5.15.8 -- Qt compiled 5.15.8
rootdir: /home/woess/workspace/git_projects/mne-python
configfile: pyproject.toml
plugins: qt-4.2.0, cov-4.1.0, timeout-2.2.0, anyio-4.0.0, harvest-1.10.4
collected 34 items

mne-python/mne/viz/_brain/tests/test_brain.py .......................... [ 76%]
........                                                                 [100%]



--- generated xml file: /home/woess/workspace/git_projects/junit-results.xml ---

============================ slowest 1 test module =============================
199.51s total  mne/viz/_brain/tests/
============================= slowest 20 durations =============================
13.46s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-vector-both]
12.85s call     mne/viz/_brain/tests/test_brain.py::test_brain_init[pyvistaqt]
11.35s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-mixed-both]
11.31s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-mixed-split]
9.82s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-mixed-lh]
9.46s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-surface-split]
9.45s call     mne/viz/_brain/tests/test_brain.py::test_brain_time_viewer[pyvistaqt]
9.44s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-surface-both]
9.40s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-vector-split]
8.74s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-vector-rh]
8.57s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-mixed-rh]
8.25s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-surface-rh]
7.61s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-vector-lh]
7.47s call     mne/viz/_brain/tests/test_brain.py::test_brain_save_movie[pyvistaqt]
6.73s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-volume-lh]
6.50s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-volume-split]
6.22s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-volume-both]
6.21s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-surface-lh]
5.70s call     mne/viz/_brain/tests/test_brain.py::test_brain_traces[pyvistaqt-volume-rh]
5.49s call     mne/viz/_brain/tests/test_brain.py::test_brain_data_gc[pyvistaqt]
======================== 34 passed in 227.22s (0:03:47) ========================
```

**Comment by larsoner** [comment]
Failures are unrelated and should be fixed in #12178

**Pull request merged**